modules/KalturaSupport/components/sourceSelector.js

```diff
@@ -312,7 +312,7 @@
 			var _this = this;
 
             var sourceLabel = this.getSourceTitle( source );
-            if( $.inArray(sourceLabel, this.sourcesList) == -1 ) {
+            if( $.inArray(sourceLabel, this.sourcesList) == -1 && source.src !== undefined ) {
```

Comment:
doesn't make a lot of sense - the issue is that we show the menu when there's adaptive stream with only one flavor.
We need to add a check for this and remove menu in this case.